src/istio/values/values.yaml

```
@@ -2,3 +2,5 @@ meshConfig:
   accessLogFile: /dev/stdout
   defaultConfig:
```

Enable Istio path normalization.
https://istio.io/latest/docs/reference/config/security/normalization/

I tested that this fixes some bypasses like `https://sso.uds.dev//admin` but have not yet tested this comprehensively.

```suggestion
  pathNormalization:
    normalization: DECODE_AND_MERGE_SLASHES
  defaultConfig:
```

At first I thought this only applied to AuthorizationPolicy path rules, but these docs mention that in addition to AuthorizationPolicies, the normalized path is used for routing and "the URL path proxied to the upstream service", so **this is perhaps as a breaking change across the entire cluster**.
https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-path_normalization

Istio best practices docs discussing normalization:
https://istio.io/latest/docs/ops/best-practices/security/#understand-path-normalization-in-authorization-policy

---

This is a pretty unusual / non-ideal behavior I think we can accept. I'll merge now, and we'll see if others disagree during the PR review